### PR TITLE
feat(full_receipt): update create items and summery items

### PR DESCRIPTION
**Pull request by jmy-kellogg**

### CODE REVIEW

File: lib/v0/pdfs/templates.full_receipts.create.request.js [hunk 1]
```diff
@@ -147,14 +147,100 @@ module.exports = {
...
+            minLength: 3,
+            maxLength: 3
+          },
+          comments: {
+            oneOf: [
+              {
+                type: 'string'
+              },
+              {
+                type: undefined
```
Comment (Endebert) [inline lib/v0/pdfs/templates.full_receipts.create.request.js]: Don't know if this is intended, but setting type here to undefined has no meaning. the resulting object would just be empty, i.e. `{}`.

I assume you want to set this to `null`?

Comment (jmy-kellogg) [reply lib/v0/pdfs/templates.full_receipts.create.request.js]: in the js code it is going to be a 'string' || undefined.
So should it be 'undefined'?

File: lib/v0/pdfs/templates.full_receipts.create.request.js [hunk 2]
```diff
@@ -147,14 +147,100 @@ module.exports = {
...
+              {
+                type: 'null'
+              }
+            ]
+          },
+          unit_price_exclude_vat: {
+            type: 'number',
+            multipleOf: 0.01
+          },
+          vat_percentage: {
```
Comment (alghanor) [inline lib/v0/pdfs/templates.full_receipts.create.request.js]: should be noted that we are talking about 19.00 as 19% and not 0.19 (as in rate)

File: lib/v0/pdfs/templates.full_receipts.create.request.js (hunk 1 above)
Comment (Endebert) [reply lib/v0/pdfs/templates.full_receipts.create.request.js]: In the end, this is supposed to be a JSON schema. Take a look at the spec here, regarding types: 

https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.1.1

> ...[type] MUST be one of the six primitive types ("null", "boolean", "object", "array", "number", or "string"), or "integer" ...

By declaring it undefined, which is javascript-specific, the end result (in JSON) would be:
```
oneOf: [
  {
    type: 'string'
  },
  { }
]
```
which may have unintended consequences (I don't know either).

I think in the end there are 3 possiblities for `comment`:
1. comment can be a string, null, or undefined
  -> allow `type: 'null'` & `type: 'string'`
2. comment can be a string, or undefined
  -> allow `type: 'string'` (undefined is covered by comment not being a required property
3. comment has to be a string
 -> `type: 'string'` & make comment `required`

File: lib/v0/pdfs/templates.full_receipts.create.request.js (hunk 2 above)
Comment (eljefedelrodeodeljefe) [reply lib/v0/pdfs/templates.full_receipts.create.request.js]: What why? Are you sure?

**Pull request merged**